x11-libs/wxGTK/wxGTK-3.0.5.1.ebuild

```diff
@@ -63,6 +63,7 @@ PATCHES=(
 	"${FILESDIR}"/wxGTK-${SLOT}-translation-domain.patch
```

Comment:
Can you revbump for this please? (git mv should be fine, tbh)

It may lead to a miscompiled binary is why.